lib/Backend/Func.cpp

```diff
@@ -376,6 +376,11 @@ Func::TryCodegen()
 {
     Assert(!IsJitInDebugMode() || !GetJITFunctionBody()->HasTry());
 
+     if (this->GetScriptContext()->IsInDebugButCantDoJITInDebug())
```

Comment:
Why are we queueing the working item and get here at all of EnableJitInDebugMode is false?

Shouldn't we not queue those instead?

Reply:
We do forceNative but forceNative and caller of this interface may miss synchronizing? Other changes also make sure that even if we miss this point the results of them will be discarded.